ui/src/heap/HeapBlock.tsx

```diff
@@ -292,11 +292,11 @@ export default function HeapBlock({
   const flag = useRouteGroup();
   const isAdmin = useAmAdmin(flag);
   const canEdit = asRef ? false : isAdmin || window.our === curio.heart.author;
-  const notEmbed = isImage && isAudio && isText && isComment;
```

Comment:
seems like this logic was faulty anyway, should have been ors right? your new logic is correct